internal/repository/pg/pg.go

```diff
@@ -0,0 +1,204 @@
+package pg
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"github.com/Painkiller675/url_shortener_6750/internal/lib/merrors"
+	"github.com/Painkiller675/url_shortener_6750/internal/models"
+	"github.com/jackc/pgerrcode"
+	"github.com/jackc/pgx"
+	"github.com/jackc/pgx/v5/pgconn"
+	_ "github.com/jackc/pgx/v5/stdlib"
+	//_ github.com/lib/pq
+
+	"time"
+)
+
+type Storage struct {
+	conn *sql.DB
+	// TODO mb use logger here
+}
+
+func NewStorage(conStr string) (*Storage, error) { // TODO: mb delete error? leave only panic
+	//connect to the database
+	conn, err := sql.Open("pgx", conStr)
+	if err != nil {
+		panic(err)
```

Comment:
```suggestion
		return err
```
И выше уже решать, что с этим делать.

Reply:
done